.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,66 @@
+name: CI Workflow
+
+on:
+  push:
+    branches: [ '**' ]
+    tags: [ '**' ]
+  workflow_dispatch:
+
+concurrency: ci-${{ github.ref }}
+
+jobs:
+  buildAndTest:
+    name: Build, Test, and Package
+    runs-on: ubuntu-latest
+    env:
+      RESOURCE_GROUP: hri-dev1-wdc-kube
+
+    steps:
+      - name: Set branch name environment variable
+        uses: nelonoel/branch-name@v1.0.1
+
+      - uses: actions/checkout@v2
+
+      - name: Copyright Check
+        run: ./copyrightCheck.sh
+
+      - name: Set up adoptopenjdk8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: 'gradle'
+
+      - name: Set GitHub Tag environment variable
+        if: startsWith(github.ref, 'refs/tags/v') == true
+        run: |
+          TAG_ID=${GITHUB_REF##*/}
+          echo "ACTIONS_TAG=$TAG_ID" >> $GITHUB_ENV
+
+      - name: Gradle build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew clean build
+
+#      TODO: Add SonarCloud (Static Code) scan after permissions issue is fixed
+#      - name: SonarCloud Scan
+#          uses: sonarsource/sonarcloud-github-action@v1.6
+#          with:
+#            projectBaseDir: ./src
+#          env:
+#            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#            SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
Left the same comment on Aram's PR -- I think this may be in a separate workflow, in which case I might remove this comment.

Reply:
I've removed the comment ✅